src/store/CategoricalDisplayStore.js

```diff
@@ -160,13 +158,15 @@ export default class CategoricalDisplayStore {
     }
 
     @computed get favoritesCategory()  {
+        this.pauseScrollSpy = true;
```

Comment:
@shayanbinary  Changing `this.pauseScrollSpy` here doesn't fix the issue.

Reply:
@morteza-binary I increase the delay, seems it works fine